apis/r/DESCRIPTION

```diff
@@ -4,7 +4,7 @@ Title: TileDB SOMA
 Description: Interface for working with 'TileDB'-based Stack of Matrices,
     Annotated ('SOMA'): an open data model for representing annotated matrices,
     like those commonly used for single cell data analysis.
-Version: 1.4.3
+Version: 1.4.4
```

Comment:
@mojaveazure Normally we'd bump the dev version here (`<major>.<minor>.<patch>.<dev>`) and save patch bumps for a release.